app/components/check_answers_summary_list_card_component.rb

```diff
@@ -28,8 +28,9 @@ def check_answers_card_title(question)
     "Person #{question.check_answers_card_number}"
   end
 
-  def action_href(log, page_id, referrer = "check_answers")
-    send("#{log.model_name.param_key}_#{page_id}_path", log, referrer:)
+  def action_href(question, log)
+    referrer = question.displayed_as_answered?(log) ? "check_answers" : nil
```

Comment:
When somebody is selecting an unanswered question, we now send them to the normal flow rather than the CYA flow.

Reply:
I'm a bit confused, sorry - what would be clicking on other than "Answer" in this scenario?

Reply:
Ah, sorry. So "change" would be the other option. When we change an answer we seem to route through any new questions that changing the answer creates.

When we click on "answer" from CYA (on unanswered question) we seem to go through the normal flow, but if I'm reading the ACs correctly, we want to only route to the unanswered questions if answering from CYA. Let me know if I'm misunderstanding or if my explanation is still confusing

Reply:
Ah, I understand the subtlety there now - good point. Before this, was there ever a scenario where the "normal" flow would display already-answered questions? If not, we could just amend the normal flow to skip unanswered questions, which would avoid creating another context a page might be viewed in to consider.